src/OpenTelemetry.Contrib.Instrumentation.Quartz/README.md

```diff
@@ -0,0 +1,14 @@
+# QuartzNET Instrumentation for OpenTelemetry .NET
+
+Automatically instruments the Quartz jobs from
+[Quartz](https://www.nuget.org/packages/Quartz/).
+
+## Installation
+
+```shell
+dotnet add package OpenTelemetry.Contrib.Instrumentation.Quartz
+```
+
```

Comment:
it'd be good to include usage example as well here.

Reply:
udpated